tests/discovery.rs

```diff
@@ -57,6 +58,11 @@ fn test() {
             .map(|iface| iface.addr.ip())
             .collect::<Vec<_>>();
 
+        let mut attributes = BTreeMap::new();
+        attributes.insert("name".to_string(), Some(format!("peer={peer_id}")));
+        attributes.insert("foo".to_string(), Some("bar".to_string()));
```

Comment:
```suggestion
        attributes.insert("føø".to_string(), Some("bär".to_string()));
```
How about some non-ASCII fun?

Reply:
:+1: